modules/developers-guide/pages/tutorials/simple-cycles.adoc

```diff
@@ -71,32 +71,32 @@ Let's take a look at a few key elements of this program:
 --
 . Save your changes and close the `+main.mo+` file to continue.
 
-== Start the local network
+== Start the local canister execution environment
 
-Before you can build the `+access_hello+` project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+Before you can build the `+access_hello+` project, you need to connect to the canister execution environment running locally in your development environment, or you need to connect toa subnet that you can access.
```

Comment:
```suggestion
Before you can build the `+access_hello+` project, you need to connect to the canister execution environment running locally in your development environment, or you need to connect to a subnet that you can access.
```

Reply:
Actually, you don't really connect to a subnet. I mean: the user's interface does not really expose those subnets to the user, right? So maybe just connect to IC blockchain mainnet?

Reply:
This whole exercise takes place locally. I think the text is generic so that they didn't have to change it depending if the tutorial is live or not

Reply:
In this case it's all local